protocol/synchronizer/synchronizer.go

```diff
@@ -98,18 +98,33 @@ func New(
 	})
 	s.eventLoop.RegisterHandler(hotstuff.ProposeMsg{}, func(event any) {
 		proposal := event.(hotstuff.ProposeMsg)
+		s.logger.Debugf("Received proposal: %v", proposal.Block)
+
+		// advance the view regardless of vote status
+		s.advanceView(hotstuff.NewSyncInfo().WithQC(proposal.Block.QuorumCert()))
+
+		pView := proposal.Block.View()
+		sView := s.state.View()
+		const alpha hotstuff.View = 10
+		if pView > sView+alpha {
+			s.logger.Infof("Dropping proposal, too high view (%v) >> state view (%v)", pView, sView)
```

Comment:
Should this be a `Warnf`? Since this is likely a Byzantine behavior.

Perhaps we should say: `Dropping proposal: proposal view too high (%v) >> replica's local view (%v)`